inst/extdata/config_schema.json

```diff
@@ -0,0 +1,191 @@
+{
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "$ref": "#/definitions/Epinow2",
+    "definitions": {
+        "Epinow2": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "job_id": {
+                    "type": "string",
+                    "format": "uuid"
+                },
+                "task_id": {
```

Comment:
I think UUIDs are perfect for task IDs

Reply:
I would also add more metadata to task ids (maybe concatenated to a UUID). So for example, the state and the as of date and the package. This has made it a lot easier for us to troubleshoot specific task failures. Again not sure this is within the scope of this PR though.